torch/csrc/lazy/core/ir_builder.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include <c10/core/ScalarType.h>
+#include <c10/util/Optional.h>
+#include <torch/csrc/lazy/backend/backend_interface.h>
+#include <torch/csrc/lazy/core/ir.h>
+#include <vector>
+
+// This file contains autogenerated LazyTensor IrBuilder
```

Comment:
probably worth changing this comment to point out that this file is part of backend interface and ops shouldn't be added without due process, except the view ops which we plan to remove soon pending functionalization